components/places/src/ffi.rs

```diff
@@ -80,6 +80,52 @@ impl PlacesApi {
         let connection = PlacesConnection { db: Mutex::new(db) };
         Ok(Arc::new(connection))
     }
+
+    // NOTE: These should be deprecated as soon as possible - that will be once
+    // we have implemented the sync manager and migrated consumers to that.
```

Comment:
maybe update this comment to something like: "they should be unused on Android but will remain needed for iOS until we can move to the sync manager there (which itself depends on replacing their existing sync engines with ours" or similar?